src/Sensors.cpp

```diff
@@ -108,6 +109,17 @@ void Sensors::setSCD30TempOffset(float offset) {
     }
 }
 
+/// set SCD30 Altitude compensation
+void Sensors::setSCD30AltitudeCompensation(float altitude) {
+    Serial.println("-->[SENSORS] SCD30 value to set altitude compensation: " + String(alt_comp));
+    if(scd30.getAltitudeCompensation() != uint16_t(alt_comp)){
+        scd30.setAltitudeCompensation(uint16_t(alt_comp));
+        Serial.println("-->[SENSORS] SCD30 setting to a new altitude value: " + String(alt_comp));
+    }
+    delay(1);
+    Serial.println("-->[SENSORS] SCD30 get altitude compensation value: " + String(scd30.getAltitudeCompensation()));
+}
+
```

Comment:
I think that this method is wrong, because we only need set the altitude compensation for the SCD30 on the scd30 init method. Only in the init method write the same validation that you did here.

Reply:
done